Dockerfile

```diff
@@ -1,7 +1,7 @@
 # syntax=docker/dockerfile:1.4
```

Comment:
does this dockerfile contain the trt-llm instructions

Reply:
trt-llm pybinding is closed source. instructions can be added in the later version as trt-llm oss has already started support pybinding with dynamic batching in v0.7.0.

Reply:
added a note in the dockerfile.